benches/criterion.rs

```diff
@@ -250,21 +250,20 @@ fn backends(c: &mut Criterion) {
                         entry_point: ep.name.clone(),
                         multiview: None,
                     };
-                    match naga::back::glsl::Writer::new(
+                    let _ = naga::back::glsl::Writer::new(
```

Comment:
I don't think that outer `let _ = ...` looks great, but I _do_ think that it's is the "least ungainly" of the options for resolving lints here, since it also resolves the inner `io::Result` that `writer.write()` produces. Consider the suggested fix when rendered fully:

```rust
                    // might be `Err` if missing features
                    if let Ok(mut writer) = naga::back::glsl::Writer::new(
                        &mut string,
                        module,
                        info,
                        &options,
                        &pipeline_options,
                        naga::proc::BoundsCheckPolicies::default(),
                    ) {
                        writer.write(); // might be `Err` if unsupported
                        //     ^^^^^^^- FIXME: warning, unused `Result`. 
                    }

```

We can't `unwrap` at either level, because that would change the semantics. Therefore, we still need to pick `drop(...)`, `let _ = ...`, or even `if let Ok(...)` _within_ the first `if let Ok(...)`. I guess I'd vote for inner `let _ = writer.write();`, if you felt that outer `if let Ok(...)` was still best (unless you feel that another option for the inner `Result` expresses intent or matches preferred style better, OFC).

I'm curious about your perspective with `let _ = ` being ungainly in the first place. What makes it so for you?

Reply:
I've applied the combined suggestions up until now (outer `if let Ok(...)` + inner `let _ = ...`) with d94fb280, but I'm still open to refinement here.

Reply:
Seems fine.